### ISSUE TITLE
Suggestions from the CK survey

### ISSUE THREAD

**Issue by gfursin** [mlcommons/ck#132]
Here are some suggestions from our [CK survey](https://cknowledge.io/questionnaire/20201020):
1) moving CK to a foundation such as LF AI or MLCommons to provide better governance and community support
2) ~~changing license to Apache 2.0~~
3) ~~dropping support for Python 2 and keeping support for Python 3.6+~~
4) rewriting CK to be more pythonic (on-going project in the CK incubator)
5) ~~including CK modules to the CK core to ensure the stability of shared workflows~~
6) making it easier to add ML workflows (providing a GUI at cKnowledge.io)
7) ~~continue automating MLPerf: https://github.com/ctuning/ck/blob/master/docs/mlperf-automation/README.md~~
8) adding more CK components and workflows for ML (BERT, DLRM): on-going
9) ~~collecting all CK components in one GitHub repository~~
10) providing better intergration with existing tools and projects including Kuberflow and PwC
11) adding super simple CK tutorials

**Comment by gfursin** [comment]
I addressed suggestions #5 and #9 in the latest CK release V1.50.0:
* https://github.com/ctuning/ck/releases/tag/V1.50.0
* https://github.com/ctuning/ai

**Comment by gfursin** [comment]
Fixed broken release: https://github.com/ctuning/ck/releases/tag/V1.55.0

**Comment by gfursin** [comment]
Some notes about the new development methodology: https://groups.google.com/g/collective-knowledge/c/0cGOPNM1Uvw

**Comment by gfursin** [comment]
Forgot to mention CLA: https://github.com/ctuning/ck/issues/135

**Comment by gfursin** [comment]
We relicensed to Apache 2.0

**Comment by gfursin** [comment]
We work with the community to revise, test, reproduce and stabilize MLPerf workflows: https://github.com/ctuning/ck/blob/master/docs/mlperf-automation/README.md
Don't hesitate to join this effort!

**Comment by gfursin** [comment]
Many of these suggestions were implemented in the latest version of the CK prototype. CK is stable and has been successfully used by different companies and organizations to automate MLPerf inference submissions. That is why we have decided not to touch it anymore expect bug fixes in the CK kernel and existing [MLOps components](https://github.com/mlcommons/ck-mlops). Instead, we are developing a new CK2 framework based on user feedback.

I close this issue.
